document/core/exec/numerics.rst

```diff
@@ -126,6 +126,7 @@ When a number is stored into :ref:`memory <syntax-mem>`, it is converted into a
 
 .. math::
    \begin{array}{lll@{\qquad}l}
+   \bytes_t(i^\ast) &=& (\bytes_t(i))^\ast \\[1ex]
```

Comment:
Is this necessary, or can you group the `bytes_` cases below using the format of the right-hand side? It seems like either case will be about as clear.

Reply:
I'm not sure what you mean by "group the bytes_ cases below using the format of the right-hand side?", can you give an example?
I wanted this case to make `bytes` take multiple numbers as arguments, and then convert them each into their bytes format.

Reply:
I just meant, can you "inline" this definition into the uses below. But honestly, I don't think it's a big deal either way.

Reply:
I will keep this here for now, I think it is useful for exec semantics too (future patch), if it ends up being unused otherwise, I can inline in. Thanks!

Reply:
Yeah, I don't understand why this definition is needed either instead of inlining the rhs. It doesn't shorten or abstract anything, and for readability you generally want to avoid the cognitive load of extra levels of definitional indirection when they are unnecessary.

Reply:
I used this definition in `bytes_i8(n^16)`, without this definition it would be `(bytes_i8(n))^16)` which I originally saw as a bit confusing, since it could be 16 copies of `bytes_i8(n)`.
I.e., given `n^16`, there are two interpretations:
1. the value that `n` represents, repeated 16 times
2. 16 non-terminal `n`, where each `n` can be different.
I guess this two cases can be clear from the context (LHS).
I'll inline this, thanks! #347